android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginScreen.kt

```diff
@@ -98,6 +99,10 @@ fun LoginScreen(loginViewModel: LoginViewModel) {
   val password by loginViewModel.password.observeAsState("")
   val loginErrorState by loginViewModel.loginErrorState.observeAsState(null)
   val showProgressBar by loginViewModel.showProgressBar.observeAsState(false)
+  val applicationConfiguration by
+    loginViewModel.applicationConfiguration.observeAsState(
+      ApplicationConfiguration(supportPhoneNumber = ""),
```

Comment:
You really don't need to init the support number to an empty string, since it already has a default